### DISCUSSION TITLE
Aideck GAP8 starting

### DISCUSSION THREAD

**Discussion by HeleMartin** [bitcraze/discussions#337] [Q&A]
Hello,

We have tried your Aideck. Xhen we flash the GAP8 on it with JTAG port and the command : ```make clean all run platform=board``` and we power-off and power-up, the GAP8 start immediately. 
We have made our own Aideck board, but when we flash it with JTAG, power-off and power-up, the GAP8 doesn't start.
How do you do to make your Aideck starts immediately ?

Thanks,

**Comment by knmcguire** [comment]
Hi! Sorry for the delay. Unfortunately since we didn't make this hardware, we won't be able to help out properly. Please communicate with Greenwaves technologies to get information about the chip and how to embed it properly.
